### ISSUE TITLE
Running the scripts with `verdi run` fails

### ISSUE THREAD

**Issue by sphuber** [aiidateam/aiida-tutorials#78]
Running the solution scripts with `verdi run` yields:
```
verdi run simple_sync_workflow.py 
Traceback (most recent call last):
  File "/home/sphuber/.virtualenvs/aiida_dev/bin/verdi", line 10, in <module>
    sys.exit(verdi())
  File "/home/sphuber/.virtualenvs/aiida_dev/local/lib/python2.7/site-packages/click/core.py", line 764, in __call__
    return self.main(*args, **kwargs)
  File "/home/sphuber/.virtualenvs/aiida_dev/local/lib/python2.7/site-packages/click/core.py", line 717, in main
    rv = self.invoke(ctx)
  File "/home/sphuber/.virtualenvs/aiida_dev/local/lib/python2.7/site-packages/click/core.py", line 1137, in invoke
    return _process_result(sub_ctx.command.invoke(sub_ctx))
  File "/home/sphuber/.virtualenvs/aiida_dev/local/lib/python2.7/site-packages/click/core.py", line 956, in invoke
    return ctx.invoke(self.callback, **ctx.params)
  File "/home/sphuber/.virtualenvs/aiida_dev/local/lib/python2.7/site-packages/click/core.py", line 555, in invoke
    return callback(*args, **kwargs)
  File "/home/sphuber/code/aiida/env/dev/aiida-core/aiida/cmdline/utils/decorators.py", line 72, in decorated_function
    return function(*args, **kwargs)
  File "/home/sphuber/code/aiida/env/dev/aiida-core/aiida/cmdline/commands/cmd_run.py", line 112, in run
    exec(compile(handle.read(), scriptname, 'exec', dont_inherit=True), globals_dict)  # yapf: disable # pylint: disable=exec-used
  File "simple_sync_workflow.py", line 11, in <module>
    from .create_rescale import create_diamond_fcc, rescale
ValueError: Attempted relative import in non-package
```
If I remove the imports of the `create_rescale` to `from .create_rescale import create_diamond_fcc, rescale` to `from create_rescale import create_diamond_fcc, rescale`, it works. @ltalirz I noticed you changed the imports to include the dot, what was the reasoning?

**Comment by ltalirz** [comment]
Hm.... sorry, that was pylint.

Can you figure out how to tell it not to do that? ;-)

**Comment by sphuber** [comment]
Fixed in PR #81
